comment/src/lib.rs

```diff
@@ -22,7 +23,7 @@ Usage
 
 Options
 
-    -m, --message               Comment message
+    -m, --message [<string>]    Provide a comment with the review (default: prompt)
```

Comment:
Note that this is the `comment` command, so it's not necessarily used with a review, hence I'd leave the description as-is.